src/App.js

```diff
@@ -3,7 +3,7 @@ import Calculator from './components/Calculator';
 
```

Comment:
- [Optional] This is a pair programming exercise, kindly consider working on this exercise tih your coding partner(s).